webapp/src/tutorial.tsx

```diff
@@ -153,16 +153,24 @@ interface TutorialCardState {
     popout?: boolean;
     showHintTooltip?: boolean;
     showSeeMore?: boolean;
+    pokeUser?: boolean;
 }
 
 export class TutorialCard extends data.Component<ISettingsProps, TutorialCardState> {
-    public focusInitialized: boolean;
     private prevStep: number;
+    private timer: number;
```

Comment:
see comment above, refactor in dedicated class -- or this will become messy soon. Eventually the hint engine state machine should be come quite complex.